packages/host/tests/acceptance/catalog-app-test.gts

```diff
@@ -392,7 +392,7 @@ module('Acceptance | Catalog | catalog app tests', function (hooks) {
         ],
       });
 
-      await waitFor('.catalog-content');
+      await waitFor('.catalog-content', { timeout: 5_000 });
```

Comment:
:thumbsup. We've been meaning to not depend on the catalog rather a mock catalog inside the test realm